src/STEMMUS_SCOPE.m

```diff
@@ -481,6 +481,9 @@
                             end
                         end
                 end
+                sfactorTot(KT) = sfactor;
+                psiLeafTot(KT) = psiLeaf;
```

Comment:
There are several issues with these two lines:
1. These two lines should be moved where `ebal` function is called, in your code after line 449. 
2. Follow the Matlab guidelines for variable names for `sfactorTot` and `psiLeafTot`. 
3. These two variables should be saved as bin as well as csv files, see [bin_to_csv.m](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/main/src/%2Bio/bin_to_csv.m), [create_output_files_binary.m](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/ddcf0b20e8dd8b1242398b876f5712946e99ab33/src/%2Bio/create_output_files_binary.m).
4. Please note that variables like `sfactor` and `psiLeaf` can be saved every time steps to binary files. It is not clear why you need to store them in other arrays i.e. `sfactorTot(KT)` and `psiLeafTo(KT)`. If you need these two arrays, they should be first defined outside of the main loop, perhaps with zero values.

Reply:
Thanks for your constructive suggestions. The `soil water stress factor` and `leaf water potential` will be saved as csv file.